tests/suites/test_suite_bignum_mod_raw.function

```diff
@@ -294,7 +294,81 @@ exit:
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+/* BEGIN_CASE */
+void mpi_mod_raw_to_mont_rep( char * input_N, char * input_A, char * input_X )
+{
+    mbedtls_mpi N, A, X;
+    mbedtls_mpi_mod_modulus m;
+
+    mbedtls_mpi_init( &N );
+    mbedtls_mpi_init( &A );
+    mbedtls_mpi_init( &X );
+
+    /* Read inputs */
+    TEST_EQUAL( 0, mbedtls_test_read_mpi( &N, input_N ) );
+    TEST_EQUAL( 0, mbedtls_test_read_mpi( &A, input_A ) );
+    TEST_EQUAL( 0, mbedtls_test_read_mpi( &X, input_X ) );
+
+    /* All of the inputs are +ve (or zero) */
+    TEST_EQUAL( 1, X.s );
+    TEST_EQUAL( 1, A.s );
+
+    mbedtls_mpi_mod_modulus_init( &m );
+    TEST_EQUAL( 0, mbedtls_mpi_mod_modulus_setup( &m, N.p, N.n, MBEDTLS_MPI_MOD_EXT_REP_BE, MBEDTLS_MPI_MOD_REP_MONTGOMERY ) );
+
+    TEST_EQUAL(0, mbedtls_mpi_mod_raw_to_mont_rep( A.p ,&m ) );
+
+    /* Calculated matches expected value */
+    TEST_ASSERT( mbedtls_mpi_cmp_mpi( &A, &X ) == 0 );
+
+   /* Output is +ve (or zero) */
+    TEST_EQUAL( 1, A.s );
```

Comment:
This can't happen (`A.s != 1`) since we test exactly the same thing at line 314, and only use `A.p` in the meantime (unless `mbedtls_mpi_cmp_mpi()` were to change its inputs, but that's not what we're testing here).

Given we don't really need an `mbedtls_mpi`, this is another argument for switching to `mbedtls_mpi_uint *`s and `mbedtls_test_read_mpi_core()`, with `ASSERT_COMPARE()` to check the output.

Reply:
It has been addressed by removing the sign checks and moving into using `mbedtls_test_read_mpi_core()`